packages/mobile/src/web3/reducer.ts

```diff
@@ -91,3 +92,5 @@ export const reducer = (
       return state
   }
 }
+
+export const userAddressSelector = (state: RootState) => state.web3.account
```

Comment:
This is a dupe of the existing currentAccountSelector: 
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/web3/selectors.ts#L3